src/components/GitStars.js

```diff
@@ -43,7 +43,7 @@ const GlyphPill = styled(Pill)`
 
 const Text = styled.div`
   padding: 0 0.325rem;
-  font-size: 13px;
+  font-size: 0.813rem;
```

Comment:
Any reason we're rounding here?

```suggestion
  font-size: 0.8125rem;
```